doc/Media2.xml

```diff
@@ -2183,7 +2183,7 @@
             <para>AudioOutputToken [tt:ReferenceToken] optional, unbounded – Optional token list of physical audio outputs. If no AudioOutputToken is provided, the device should play the audio clip through all physical audio outputs.</para>
           </listitem>
           <listitem>
-            <para>Type [xs:string] - Format of the audio clip. For definitions see tr2:SupportedAudioClipFormat parameter.</para>
+            <para>Type [xs:string] - Format of the audio clip. Supported audio Clip formats are WAV, MP3, OGG-Opus, OGG-Vorbis, OGG-Flac, OGG-Speex.</para>
```

Comment:
Should we define these strings OR refer to the IANA like we did for Audio/Video Encoding?

Reply:
Preferred to list individual strings for this feature.

Reply:
Should't we use IANA mime types or names for the formats if possible? 
E.g. audio/ogg;codecs=opus, audio/ogg;codecs=flac etc?
https://www.rfc-editor.org/rfc/rfc7845.html#section-9